docs/en/pact-reference.md

```diff
@@ -1378,28 +1378,16 @@ reference to the Kadena `coin` KDA token module, because `coin` implements
 can specify any defined interface and accept any module that implements said
 interface.
 
-In this way, modrefs bring polymorphism to Pact. To programmers coming from
-polymorphic languages like Javascript, Java or Python, module references may
-look familiar. However, Pact is very much not object-oriented so there are
-important differences in how polymorphism works and what it is best used for.
-
-A key difference is that while modrefs allow polymorphism over modules, they
-don't offer polymorphism over data. Modules in Pact are not data in any
-practical sense: not only are they "code only", but they cannot be dynamically
-created within the Pact system.
-
-Meanwhile, data stored in the database is managed using schemas, which have no
-polymorphic features via modrefs or any other mechanism. This is why we suggest
-modrefs are more about "interoperation" instead of polymorphism per se, avoiding
-the object-oriented connotation of "objects changing shape". Modrefs allow modules to
-"interoperate with each other".
-
-Additionally, programmers should resist the urge to employ other object-oriented
-patterns using modrefs. A popular idea of "coding around interfaces, not
-implementations" is actually harmful in Pact if polymorphism is not
-needed. Programmers should prefer direct references whenever possible as they
-are not only faster but safer, as Pact aggressively and permanently inlines all
-directly-referenced code in the interests of code stability.
+The polymorphism offered by modrefs resembles Generics in Java or Traits in Rust,
+and should not be confused with more object-oriented polymorphism like that found
+with Java classes or Typescript types. Modules cannot "extend" one another, they
```

Comment:
TypeScript is spelled with a capital 'S' (see [the language homepage](https://www.typescriptlang.org/))

```suggestion
with Java classes or TypeScript types. Modules cannot "extend" one another, they
```